glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt

```diff
@@ -244,6 +246,44 @@ class GleanTest {
         }
     }
 
+    @Test
+    fun `test sending of startup baseline ping`() {
+        // Set the dirty flag.
+        LibGleanFFI.INSTANCE.glean_set_dirty_flag(true.toByte())
+
+        // Restart glean and don't clear the stores.
+        val server = getMockWebServer()
+        val context = getContextWithMockedInfo()
+        resetGlean(context, Glean.configuration.copy(
+            serverEndpoint = "http://" + server.hostName + ":" + server.port,
+            logPings = true
+        ), false)
+
+        try {
+            // Trigger worker task to upload the pings in the background
+            triggerWorkManager(context)
+
+            val request = server.takeRequest(20L, TimeUnit.SECONDS)
+
+            val baselineJson = JSONObject(request.body.readUtf8())
+            assertEquals("baseline", baselineJson.getJSONObject("ping_info")["ping_type"])
```

Comment:
This is going away (#653). Over there we're checking the URL for the right type now.
(That patch is currently stuck, I should get on to it).

If you already adopt the other way to check this then we won't run into merge problems one way or another.